azure-communication-ui/build.gradle

```diff
@@ -47,8 +47,8 @@ buildscript {
 
             // In this branch, it was already Beta version, so it'll be in the GA field here
             // This should be set to a GA release before merge back
-            azure_calling_sdk_version_ga = '2.6.0'
-            azure_calling_sdk_version_beta = '2.6.0'
+            azure_calling_sdk_version_ga = '2.5.0'
+            azure_calling_sdk_version_beta = '2.5.0'
```

Comment:
Please keep _beta at the newest released version (or beta) always. 

The beta build uses API in 2.6.0-beta+ so this change will break the beta build features (access to getLogFiles()) that have compile time features implemented.